src/query/string_opfunc.c

```diff
@@ -3189,8 +3202,10 @@ db_json_insert (DB_VALUE * result, DB_VALUE * arg[], int const num_args)
     case DB_TYPE_JSON:
       new_doc = db_json_get_copy_of_doc (arg[0]->data.json.document);
       break;
-
-      /* TODO - confirm DB_TYPE_NULL */
+    case DB_TYPE_NULL:
+      new_doc = db_json_allocate_doc ();
```

Comment:
newly allocated json is by default null. calling db_json_make_document_null seems redundant.